src/components/PromoSheet/PromoSheet.scss

```diff
@@ -0,0 +1,68 @@
+@use '../variables';
+
+$block: '.#{variables.$ns}promo-sheet';
+
+#{$block} {
+    &__content[class] {
+        width: auto;
+        padding: var(--yc-promo-sheet-padding);
+        margin: 0 var(--yc-promo-sheet-margin) var(--yc-promo-sheet-margin);
+
+        color: var(--yc-promo-sheet-foreground);
+        background: var(--yc-promo-sheet-background);
+        border-radius: var(--yc-promo-sheet-border-radius);
+    }
+
+    &__header {
+        position: relative;
+
+        padding: 0 20px 0 0;
+        margin: 0 0 var(--yc-promo-sheet-header-margin);
+    }
+
+    &__title {
+        margin: 0;
+
+        font-size: var(--yc-text-header-1-font-size);
+        line-height: var(--yc-text-header-1-line-height);
+    }
+
+    &__close-button {
+        position: absolute;
+        top: calc(12px * -1);
+        right: calc(12px * -1);
+    }
+
+    &__message {
+        margin: 0 0 var(--yc-promo-sheet-message-margin);
+
+        font-size: var(--yc-text-body-3-font-size);
+        line-height: var(--yc-text-body-3-line-height);
+    }
+
+    &__image-container {
+        margin-bottom: var(--yc-promo-sheet-image-margin);
+    }
+
+    &__image {
+        display: block;
+
+        width: 100%;
+        height: auto;
+    }
+
+    &__action-button {
+        display: block;
+    }
+}
+
+.yc-root {
+    --yc-promo-sheet-margin: 8px;
+    --yc-promo-sheet-padding: 20px;
+    --yc-promo-sheet-border-radius: 12px;
+    --yc-promo-sheet-header-margin: 12px;
+    --yc-promo-sheet-message-margin: 16px;
+    --yc-promo-sheet-image-margin: 12px;
+    --yc-promo-sheet-foreground: var(--yc-color-text-light-primary);
+    --yc-promo-sheet-background: var(--yc-my-color-brand-normal);
```

Comment:
Same question here, we would allow to style this component?

Reply:
Yes